frontend/src/pages/Footer/_Footer.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./_Footer.scss";
+import { Button } from "../../components/Buttons/Buttons";
+
+const Footer = ({ menu, logoDesktop, logoMobile, logoFooter }) => {
+  const Logo = () => {
+    return (
+      <div className="inner-one">
+        <a href="/">
+          <img className="logo-desktop-footer" src={logoDesktop} />
+          <img className="logo-mobile-footer" src={logoMobile} />
+        </a>
+      </div>
+    );
+  };
+
+  return (
+    <div className="footer-nav flex-container ">
+      <Logo />
+      <div className="footer-menu">
+        {menu.map((item, idx) => {
+          return (
+            <React.Fragment key={idx}>
```

Comment:
`<Fragment>` is already imported

Reply:
When I use  <> </>  and assign it a key, my style breaks. I do not want to use a <div> as I have to restyle my CSS.  That is why i used <React.Fragment> and assigned it a key. I googled the issue and it was recommended , that to add a key to a  fragment one needs to use full Fragment syntax.

Reply:
I actually meant importing fragment as

```
import React, { Fragment } from "path/to/react"
```

but looking back at the import line, it seems `<Fragment>` is not already imported. I must have been looking at the wrong file. Sorry for the mistake. Feel free to import Fragment as a destructured object, if you want. In either case, this is resolved.